frontend/src/components/main-header.js

```diff
@@ -48,6 +48,12 @@ class MainHeader extends React.Component {
       margin: 0 5rem 0.5rem 0;
       font-size: 0.875rem;
     }
+
+    .autosave::before {
```

Comment:
i dont think this is necessary because we have the help icon telling them that its required plus this is a common UI pattern. if we decide to keep it though, can you place it on the same line as 'your progress is autosaved'? (just make them 2 sentences to as separation)